frontend/src/app/auth/auth.service.ts

```diff
@@ -5,21 +5,34 @@ import { Router } from '@angular/router';
 import { filter } from 'rxjs/operators';
 import * as auth0 from 'auth0-js';
 import { CookieService } from 'ngx-cookie-service';
+import { JwtHelperService } from '@auth0/angular-jwt';
+import { User } from '../models/User';
+
+
+
```

Comment:
These files should be formatted with prettier.

Reply:
also a lot of unnecessary whitespace all over this file

Reply:
Formatted with prettier, fixed it :)